Doc/whatsnew/3.14.rst

```diff
@@ -153,10 +153,10 @@ As another example, generating HTML attributes from data:
 .. code-block:: python
 
    attributes = {"src": "shrubbery.jpg", "alt": "looks nice"}
-   template = t"<img {attributes} />"
-   assert html(template) == '<img src="shrubbery.jpg" alt="looks nice" class="looks-nice" />'
+   template = t"<img {attributes}>"
```

Comment:
In XHTML and HTML5, you should close the `<img>` tag, no? https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/img

You might propose a separated PR to change PEP 750 text.

Reply:
HTML5:

> _Self-closing tags (`<tag />`) do not exist in HTML._

XHMTL:

> Self-closing tags are required in void elements in [XML](https://developer.mozilla.org/en-US/docs/Glossary/XML), [XHTML](https://developer.mozilla.org/en-US/docs/Glossary/XHTML), and [SVG](https://developer.mozilla.org/en-US/docs/Glossary/SVG) (e.g., `<circle cx="50" cy="50" r="50" />`).

(`img` is a void element: it must have a start tag `<img>` but must not have an end tag `</img>`.)

https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags

But this example is HTML and we can ignore XHTML.

Reply:
Oh ok, I trust you. I didn't write HTML for many years :-)